### PR TITLE
(FM-5815) Prep for 1.3.1 release

### PR DESCRIPTION
**Pull request by branan**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -11,6 +11,7 @@ group :test do
   gem 'metadata-json-lint', '~> 0.0'
   gem 'rspec-puppet-facts', '~> 1.3'
   gem 'semantic_puppet', '0.1.3'
+  gem 'puppet-blacksmith', '>= 3.4.0',      :require => false, :platforms => 'ruby'
```
Comment (MikaelSmith) [inline Gemfile]: This looks like it might be a problem for Travis.

Comment (MikaelSmith) [inline Gemfile]: See https://travis-ci.org/puppetlabs/puppetlabs-puppet_agent/builds/176248555. We should use a PR for merge-up to master to ensure that gets fixed.

Comment (branan) [inline Gemfile]: Ugh, yup. I'll re-open this without the blacksmith change - we can get it on to master after. This was sort of a preemptive change.

Comment (branan) [inline Gemfile]: Oh wait, this is merged, not closed. Nevermind, then!

**Pull request merged**